### ISSUE TITLE
Proj0219 false positive

### ISSUE THREAD

**Issue by sungam3r** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#631]
```
The <IsPackable> node is set to 'true' but the <OutputType> is not 'Library' (https://dotnet-project-file-analyzers.github.io/rules/Proj0219.html)
```

This one is FP for .NET tools (.NET global/local tools). See https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create.

OutputType for such tools is `Exe`.

Examples:
- https://github.com/danielpalme/ReportGenerator/blob/main/src/ReportGenerator.DotnetGlobalTool/ReportGenerator.DotnetGlobalTool.csproj#L4
- https://github.com/PublicApiGenerator/PublicApiGenerator/blob/master/src/PublicApiGenerator.Tool/PublicApiGenerator.Tool.csproj#L4
